### PR TITLE
Feat/graph 7738 review card component

### PR DESCRIPTION
**Pull request by rauldiazpoblete**
[FIGMA](https://www.figma.com/file/1rUpciJJRVQZiNqEzggaAN/branch/8S4JRk3JMuPc0ljbyUoN7C/Design-System?node-id=5498%3A97219)

- Update Card background colors
- Add different border styles for hover & kinds active & danger
- Add a disabled style

### CODE REVIEW

File: packages/core/src/common/_customs.scss [hunk 1]
```diff
@@ -142,13 +142,13 @@ $g-dark-base3: hsl(225, 14%, 19%);
...
-$g-base3: hsl(228, 38%, 97%);
-$g-base4: hsl(0, 0%, 98%);
-$g-base5: hsl(0, 0%, 100%);
+$g-base0: hsl(0, 0%, 100%);
+$g-base1: hsl(240, 20%, 99%);
+$g-base2: hsl(195, 20%, 96%);
+$g-base3: hsl(225, 22%, 96%);
+$g-base4: hsl(231, 26%, 95%);
+$g-base5: hsl(227, 18%, 90%);
+$g-base6: hsl(200, 86%, 97%);
```
Comment (LunaFidalgo) [inline packages/core/src/common/_customs.scss]: These colors are related to the "light" theme. I think that maybe we should change the name to: `g-light-baseX`. I'm working in graphext on the light/dark theme and I added these like this (we still need to figure out how we are going to share them between graphext & BP). 
```
	/* base light */
	--g-light-base0: hsl(0deg, 0%, 100%);
	--g-light-base1: hsl(240deg, 20%, 99%);
	--g-light-base2: hsl(195deg, 20%, 96%);
	--g-light-base3: hsl(225deg, 22%, 96%);
	--g-light-base4: hsl(231deg, 26%, 95%);
	--g-light-base5: hsl(227deg, 18%, 90%);
	--g-light-base6: hsl(200deg, 86%, 97%);
```

File: packages/core/src/components/card/_overrides.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  &.#{$ns}-kind-g-active,
+  &.#{$ns}-kind-g-active:hover {
+    border-color: $g-primary3;
+  }
+
+  &.#{$ns}-kind-g-error,
+  &.#{$ns}-kind-g-error:hover {
+    border-color: $g-error-600;
```
Comment (LunaFidalgo) [inline packages/core/src/components/card/_overrides.scss]: We are adding the custom classes in this [classes file](https://github.com/graphext/blueprint/blob/graphext/packages/core/src/common/classes.ts#L382). It helps to know what custom classes we have in each component and make it easier to use them in graphext.

File: packages/core/src/components/card/_overrides.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    &.#{$ns}-interactive:hover {
+      box-shadow: none;
+      cursor: auto;
+    }
+  }
+
+  &.#{$ns}-kind-g-active,
+  &.#{$ns}-kind-g-active:hover {
+    border-color: $g-primary3;
```
Comment (LunaFidalgo) [inline packages/core/src/components/card/_overrides.scss]: In the design, they specify that the active stoke color is the primary700:
image

The primary colors are not added yet but if this design needs them, I think we should add them here!

**Review by LunaFidalgo — CHANGES_REQUESTED** [review]

File: packages/core/src/components/card/_overrides.scss (hunk 3 above)
Comment (diegographext) [reply packages/core/src/components/card/_overrides.scss]: Now the primary color is added. Is only required to do a `merge` with `graphext` branch 😄

File: packages/core/src/common/_customs.scss (hunk 1 above)
Comment (rauldiazpoblete) [reply packages/core/src/common/_customs.scss]: Done with [ca7f370](https://github.com/graphext/blueprint/pull/61/commits/ca7f37046f7e28ff2db81f63f5918e994f4fbbbe)

File: packages/core/src/components/card/_overrides.scss (hunk 2 above)
Comment (rauldiazpoblete) [reply packages/core/src/components/card/_overrides.scss]: Right! Done with [87c8390](https://github.com/graphext/blueprint/pull/61/commits/87c83900f543de32e6510f4ff85edc2bbfee3dd6)

File: packages/core/src/components/card/_overrides.scss (hunk 3 above)
Comment (rauldiazpoblete) [reply packages/core/src/components/card/_overrides.scss]: Added with [b1f8398](https://github.com/graphext/blueprint/pull/61/commits/b1f83981aab938d6ee56f34907f6c7a4a4b81d8c)

**Review by LunaFidalgo — APPROVED** [review]

**Pull request merged**